packages/validator/src/services/validatorStore.ts

```diff
@@ -31,26 +31,37 @@ import {List, toHexString} from "@chainsafe/ssz";
 import {routes} from "@chainsafe/lodestar-api";
 import {ISlashingProtection} from "../slashingProtection";
 import {BLSKeypair, PubkeyHex} from "../types";
-import {getAggregationBits, mapSecretKeysToValidators} from "./utils";
+import {getAggregationBits, mapSecretKeysToValidators, mapPublicKeysToValidators, requestSignature} from "./utils";
+import {SignerType, Signers} from "../validator";
 
-/**
- * Service that sets up and handles validator attester duties.
- */
```

Comment:
can you restore this comment?

Reply:
Yeah of course, must have been accidental. 
https://github.com/ChainSafe/lodestar/pull/3502/commits/a0dca7494d5d4a12efd2449963151b3a00971a67